src/sentry/static/sentry/app/views/organizationIntegrations/addIntegrationButton.jsx

```diff
@@ -31,6 +31,7 @@ export default class AddIntegrationButton extends React.Component {
   static propTypes = {
     provider: PropTypes.object.isRequired,
     onAddIntegration: PropTypes.func.isRequired,
+    reinstall: PropTypes.bool,
```

Comment:
Let's have this take a `text` property instead, it will be a wee bit more generic, but then it will render the text if passed as a prop, otherwise it will render `buttonText`.